sender/screenshare.py

```diff
@@ -30,12 +30,13 @@ def icon(self) -> str:
         return "video-display"
 
     def parameters(self) -> Tuple[int, int, int]:
-        monitor = Gdk.Display().get_default().get_monitor(0)
-        scale, geometry = monitor.get_scale_factor(), monitor.get_geometry()
-        return (scale * geometry.width, scale * geometry.height, 30, {})
+        monitor = Gdk.Display().get_default().get_monitor(0).get_geometry()
+        return (monitor.width, monitor.height, 30, {})
 
     def pipeline(self, width: int, height: int, fps: int,
                  **kwargs) -> List[str]:
+        monitor = Gdk.Display().get_default().get_monitor(0).get_geometry()
+        screen = Gdk.Screen().get_default()
```

Comment:
With getting geometry again you risk getting different result, if resolution changed in the meantime for example.
You can avoid it passing it through kwargs. See how webcam uses it (for a different purpose): https://github.com/QubesOS/qubes-video-companion/commit/9bb6481bbf2dfdd992774957fa10a8bae1237080#diff-4ac1eb78dde67ed6f3f8d985cbeba4acdde0049b79411f69465f489fe3389bb3R66

Reply:
makes sense, fixed!